content/elections-chatbot.md

```diff
@@ -0,0 +1,29 @@
++++
+internal-id = "ELEXCHATBOT"
+kicker = "Elections"
+title = "Your 2024 Election Assistant"
+title-tag = "Spotlight PA’s 2024 Election Assistant"
+description = "Have questions about the 2024 election in PA? Ask our AI-powered assistant and get a trusted answer based on Spotlight PA’s election coverage."
+dek = "Candidates for Pennsylvania Supreme Court, Commonwealth Court, and more will appear on the Nov. 7 ballot."
+linktitle = "Spotlight PA’s 2024 Election Assistant"
+blurb = "Have questions about the 2024 election in PA? Ask our AI-powered assistant and get a trusted answer based on Spotlight PA’s election coverage."
+url = "/elections/chatbot"
+layout = "page-landing"
+type = "elections"
+no-index = true
++++
+## Have questions about the 2024 election? Ask our AI-powered assistant. Get trusted answers and resources to learn more.
+
+{{<width-screen>}}
```

Comment:
I would call it “dewey-embed” instead of “width-screen” because it isn’t a general screen width helper, just one for Dewey.